experimental/devices/inky/inky.go

```diff
@@ -292,6 +289,9 @@ func (d *Dev) update(border byte, black []byte, red []byte) (err error) {
 }
 
 func (d *Dev) reset() (err error) {
+	if err = d.busy.In(gpio.PullUp, gpio.FallingEdge); err != nil {
```

Comment:
That was an error in the code?

Reply:
Err I actually can't remember why I moved this :smile: 
I've moved it back - it seems to work either way.